### ISSUE TITLE
Transition from intro to body

### ISSUE THREAD

**Issue by mhosken** [usfm-bible/tcdocs#12]
Currently, the USX specification has a strong ordering requirement for introductory paragraphs and then requires a chapter marker to precede any chapter content (or body content) like normal paragraphs and verses. This is problematic in modules, where there is often no chapter markers. I would suggest therefore that we remove the requirement for chapter content to be preceded by a chapter marker and instead say that once you hit the first chapter content, then you are done with the introductory paragraphs. Thus you can have \ip \p rather than requiring \ip \c\ p.

**Comment by KentSpiel** [comment]
In Biblica We have a standard that introductory material always begins\imt and ends with\ie. \imt Is not publishable and is followed by a bit of meta-data, namely the word or abbreviation for "Introduction" that should be used in a electronic table of contents. There is a bit of history behind this. I adopted this based on the recommendation from Mark Howe in 2015 when he was still working for YouVersion. He said he needed explicit markers for the beginning and ending of the introduction.

So I would suggest that if you have introductory material the \ie should be required. If You have introductory material and you reach \c then you can add one implicitly. 
Maybe you can say introductory material ends when either you reach \ie or \c .

**Comment by klassenjm** [comment]
There is a context in USFM implied by the first letter of some markers - `\i`, `\f`, `\x`, `\e`. It's not perfectly consistent (e.g. there are markers like `\mt#` which occur before `\c`). Sticking with this basic protocol for introductory context inevitably meant introducing a bunch of new introductory markers starting with `\i` because some projects wished to have lists or poetic texts in in the introduction.

I'm wondering whether the proposal from Martin suggests that it would be reasonable to abandon an expectation that introductory, non-scripture text, needs to be marked with markers beginning with `\i`. This would mean reducing the number of markers in the USFM spec by quite a bit, eventually. Alternatively, perhaps the collection of `\i` markers remains and can all be used flexibly within the scripture body area, but with the clear knowledge that _this is not scripture text_.

Perhaps what my reply is asking is -- is it helpful to have clarity about the context you're in, and whether this is scripture text content or not, and how do we retain that awareness?

Would it also be an option for have an alternative path in the schema for "Module", like there is for Scripture and Peripheral? Perhaps that's a poor design. I'm thinking this because there are different content in which USFM / USX is used (Bible book, peripheral, module etc.)

**Comment by mhosken** [comment]
Today a sufficient solution is to use `\ie`. We might well look into this in the future as to how best to transition from introductory material to body text and what markers to allow in the introduction.

**Comment by mhosken** [comment]
I don't think we want to get away from contrasting introductory material from main text material, so I'm not thinking we do away with `\i` for intro type things. The contrast is important in scripture text (e.g. going from single to double column). But it may be less so for modules which don't really have introductory text per se.

**Comment by mhosken** [comment]
One reason for wanting to get rid of the required \c in a Scripture element is that we want to be able to pass around Scripture fragments (which are basically a Scripture with no \id line / BookIdentification). For example in response to a \ref in a module. A fragment is simply a list of paragraphs (in appropriate order, etc.). There are other additions we could make for such fragments (e.g. add an optional @vid to <usx> and a corresponding expression in usfm).

As it stands we could just remove the required Chapter element from the Scripture definition. This would have the effect of making the introductory material greedy so any initial tables would be treated as intro material, etc. I don't think this is going to be a problem.
